sdk/identity/identity-broker/samples/v1-beta/typescript/src/index.ts

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * @summary Demonstrates usage of @azure/identity-broker and @azure/identity
+ * packages for WAM MSA support.
+ */
+
+import {
+  InteractiveBrowserCredential,
+  InteractiveBrowserCredentialNodeOptions,
+  useIdentityPlugin,
+} from "@azure/identity";
+import { nativeBrokerPlugin } from "@azure/identity-broker";
+import dotenv from "dotenv";
+import { app, BrowserWindow } from "electron";
+const win = new BrowserWindow();
+
+useIdentityPlugin(nativeBrokerPlugin);
+
+// Load the environment
+dotenv.config();
+
+function createWindow() {
+  const win = new BrowserWindow({
+    width: 800,
+    height: 600,
+  })
+  win.loadFile('index.html')
+  return win.getNativeWindowHandle()
+}
+
+app.on('ready', async () => {
+  let winHandle = createWindow()
+
+  app.on('activate', () => {
+    if (BrowserWindow.getAllWindows().length === 0) {
+      winHandle = createWindow()
+    }
+  })
+try{
+
+  const options: InteractiveBrowserCredentialNodeOptions = {
+    clientId: process.env.AZURE_CLIENT_ID || "client",
+    authorityHost: process.env.AZURE_AUTHORITY_HOST,
+    tenantId: process.env.AZURE_TENANT_ID,    
+    brokerOptions: {
+      enabled: true,
+      legacyEnableMSAPassthrough: true,
```

Comment:
Q: Why the inconsistency in casing here? I'm looking at `tenantId` vs. `legacyEnableMSAPassthrough` in this block of code. I'd expect the property to use the casing `legacyEnableMsaPassthrough`.